Library/Homebrew/cmd/fetch.rb

```diff
@@ -42,9 +42,15 @@ def fetch
   end
 
   def fetch_bottle?(f)
-    return true if ARGV.force_bottle? && f.bottle
-    return false unless f.bottle && f.pour_bottle?
-    return false if ARGV.build_from_source? || ARGV.build_bottle?
+    return false unless f.bottle
+    return true if ARGV.force_bottle?
```

Comment:
This is incorrect. If a formula has not bottle, it shouldn't return true event if force bottle flag is passed.